src/lavinmq/vhost.cr

```diff
@@ -300,12 +300,12 @@ module LavinMQ
         when AMQP::Frame::Exchange::Bind
           src = @exchanges[f.source]? || return false
           dst = @exchanges[f.destination]? || return false
-          src.bind(dst, f.routing_key, f.arguments.to_h)
+          return false unless src.bind(dst, f.routing_key, f.arguments.to_h)
           store_definition(f) if !loading && src.durable && dst.durable
```

Comment:
Doesn't `false` indicate that the client will get a PreconditionFailed back? as in the case above if the exchange is missing? Should we just not store the defintion instead? 

```suggestion
          if src.bind(dst, f.routing_key, f.arguments.to_h)
            store_definition(f) if !loading && src.durable && dst.durable
          end
```

Reply:
Maybe :) It wasn't obvious on what the return value meant (or how it's used). At least not used in `Client`. Maybe we should use enums or symbols here and there instead to get more descriptive return values?

Reply:
I think i chose to return false because we did so for `Queue::Declare` if the queue already existed.